app/Http/Controllers/WebhookController.php

```diff
@@ -92,6 +98,13 @@ private function push(): string
         $project = Project::firstWhere('gitlab_project_id', request('project.id'));
         abort_if($project == null, 404);
 
+        if ($this->isBranchDeletionPush())
```

Comment:
I think we need a meeting to talk about these fixes.
Why asking if "isBranchDeletionPush", instead of asking for the one we want to process and skip all the other ones that we are no interested?

Reply:
We can't gitlab sends everything in a push trigger, so it's our job to filter out what we don't want, and that way is simpler since we want more or less 99%, but that 1% we don't want is filtered out with this check here.

I will however add a test for this.